src/System Application/App/AppSource Gallery/src/IAppSourceProductManagerDependencies.Interface.al

```diff
@@ -0,0 +1,38 @@
+
+// ------------------------------------------------------------------------------------------------
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+// ------------------------------------------------------------------------------------------------
+namespace System.Apps.AppSource;
+
+using System.Environment.Configuration;
+using System.RestClient;
+
+/// <summary>
+/// Interface for managing dependencies to the AppSource Product Manager codeunit.
+/// </summary>
+interface "IAppSource Product Manager Dependencies"
```

Comment:
Why an interface? It's internal so no one else can use it. Are we expecting to create multiple implementations? 

I assume you did this to get a test implementation to mock the real thing. That would require you to expose internals. The way we do that is we expose them to a Test Library, the test library makes public what is needed in order to test. 

Here i see no test library nor that the internals have been exposed.

Reply:
Is there an interface naming standard? The other System App interface names don't start with an `I`.

Reply:
Next PR